src/man/sssd.conf.5.xml

```diff
@@ -3406,6 +3406,46 @@ pam_gssapi_indicators_map = sudo:pkinit, sudo-i:pkinit
                     </listitem>
                 </varlistentry>
 
+                <varlistentry>
+                    <term>dns_resolver_server_timeout (integer)</term>
+                    <listitem>
+                        <para>
+                            Defines the amount of time (in milliseconds)
+                            SSSD would try to talk to DNS servers before
```

Comment:
Little wording change: `to a given DNS server before`

Reply:
I have rephrased the complete sentence. Can you pls check the latest changes. 

Also I have split the dns_resolver_* parameters from  “failover.xml” and created a new file “dns_options.xml” to sync the parameter definition since currently the definition for the parameter “dns_resolver_timeout” is different in files “failover.xml” and “sssd.conf.5.xml”. 

It will help maintaining the dns_resolver_* parameters definition from a single file.